runners/core-construction-java/src/test/java/org/apache/beam/runners/core/construction/ValidateRunnerXlangTest.java

```diff
@@ -135,6 +172,15 @@ public void multiInputOutputWithSideInputTest() {
     PAssert.that(pTuple.get("side")).containsInAnyOrder("ss");
   }
 
+  /**
+   * Motivation behind groupByKeyTest.
+   *
+   * <p>Target transform – GroupByKey
+   * (https://beam.apache.org/documentation/programming-guide/#groupbykey) Test scenario – Grouping
```

Comment:
Move "Test scenario" to a new line here and below.

Reply:
Ditto as my comment earlier. I'll see how to resolve this.